scripts/tools.js

```diff
@@ -17,7 +17,7 @@ module.exports.printOpts = (key, value) =>
       : `="${value}"`
   }`;
 
-module.exports.lernaRun = (command, components) => `lerna -- run ${command} --no-bail --stream --include-dependencies ${
+module.exports.lernaRun = (command, components) => `lerna -- run ${command} --no-bail ${command === "build" ? `--stream` : `--parallel`} --include-dependencies ${
```

Comment:
Looks fine but this command is getting mighty long. May be worth pulling this ternary out to it's own line.

Reply:
+1 Mind if we tackle that in another issue?